ForgetItNot/settings.py

```diff
@@ -38,6 +39,8 @@
     'flashcards',
     'users',
     'rest_framework',
+    'rest_framework.authtoken',
+    'djoser',
```

Comment:
Just as a "good practice", you should order the `INSTALLED_APPS`. Start with `django` apps, then `rest_framework` ones, then installed packages, then your own.